crispy_forms_materialize/templates/materialize_css_forms/field.html

```diff
@@ -54,6 +56,7 @@
 
     {% if field|is_checkbox %}
     </{% if tag %}{{ tag }}{% else %}p{% endif %}>
+    {% elif field|is_checkboxselectmultiple %}
```

Comment:
Not sure about this too, an empty elif statement.

Reply:
I was not sure if a custom tag should be handled in is_checkboxselectmultiple case. The empty case is there just as a reminder. Can be safely removed.